maps/clarion.dmm

```diff
@@ -44257,10 +44257,6 @@
 /obj/item/storage/goodybag,
 /obj/item/peripheral/videocard,
 /obj/item/storage/pill_bottle/cyberpunk,
-/obj/item/aiModule/hologram_expansion/clown{
```

Comment:
?

Reply:
Hmm, I may have misclicked one or twice in the editor, or I may be using an outdated version of the map, which would be strange...